src/components/Button/Button.scss

```diff
@@ -0,0 +1,46 @@
+@import '../../variables.module.scss';
+
+.copernicus-button {
+  padding: 1px 2px;
+  height: 35px;
+  min-width: 109px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+
+  &.primary {
+    background-color: $whiteBackground;
+    color: $primaryColor;
+    border: 1px solid $primaryColor;
+
+    &:hover {
+      background-color: $dropdownHoverBackground;
+    }
+  }
+
+  &.danger {
+    background-color: $errorCol;
+    color: $textColor;
+
+    &:hover {
+      background-color: $dropdownHoverBackground;
```

Comment:
I didn't find anything about hovering in figma so i just used the one from the dropdown.

Reply:
I got an answer from Kristijan.

The green button should behave the same as in `Search` tab (it gets a bit darker)

The blue button should on hover get blue background and white text